### ISSUE TITLE
LC0039 and Fully Qualified Object Name

### ISSUE THREAD

**Issue by tscottjendev** [StefanMaron/BusinessCentral.LinterCop#386]
Getting warning, similar to #384 and #380 but a little different

> Argument 2: cannot convert from 'Record "Job Journal Line"' to 'Record "Job Journal Line"'.

When passing a fully qualified variable to Page.Run

```
    internal procedure Foo()
    var
        JobJournalLine: Record Microsoft.Projects.Project.Journal."Job Journal Line";
    begin
        Page.Run(Page::"Custom Job Journal", JobJournalLine);  //Page is our custom page
    end;

```

**Comment by Arthurvdv** [comment]
Interesting 🤔

Could you help me with a repro? This small example below seems to work, so probarly something with the page object what I'm missing here.


```AL
codeunit 50100 MyCodeunit
{
    internal procedure Foo()
    var
        JobJournalLine: Record Microsoft.Projects.Project.Journal."Job Journal Line";
    begin
        Page.Run(Page::"Custom Job Journal", JobJournalLine);
    end;
}

page 50100 "Custom Job Journal"
{
    ApplicationArea = All;
    PageType = Worksheet;
    UsageCategory = None;
    SourceTable = "Job Journal Line";
}
```

**Comment by Arthurvdv** [comment]
Ah, that kinda makes sense.

I still would love the have a repo to look into if I could improve the rule on this, because the message you're receiving now is a bit confusing.

Probably still overlooking something here, but this doesn't raise the rule0039 for me 🤔

![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/44637996/a3e9307c-0711-4991-b863-e42f3206d490)

**Comment by Arthurvdv** [comment]
I couldn't find a repo on this, but I think I have resolved this error.

Hoping that this is resolved in the updated (pre)release version of [v0.30.8](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.30.8).
